server/src/routes/auth.ts

```diff
@@ -178,5 +185,75 @@ route.get('/logout', async function (req: Request, res: Response) {
         });
     }
 });
+// use this route to implement the update seen greating. Link that to the skip button and the user will only see the form once
+route.put(
+    '/greetingModal/:_id/boarded',
+    async function (req: Request, res: Response) {
+        if ((req.session as ISession)._id) {
+            const _id = req.params._id;
+            const updated = updateSeenGreeting(_id);
+
+            res.status(200).json({ message: 'Value updated' });
+        }
+    }
+);
+// generating exercises automatically
+route.get('/createExercises', async function (req: Request, res: Response) {
+    await createExercises();
+    return res.status(200).json({ message: 'Value updated' });
+});
+
+//users adding their own exercises
+route.post('/addExercise/:_id', async function (req: Request, res: Response) {
```

Comment:
Pass name of user instead of id since frontend does not have access to MongoDB ObjectId on the User object (name is also unique and can be used to locate a User)